src/tests/bundles/bionic-queens.yaml

```diff
@@ -0,0 +1,142 @@
+series: bionic
+
+applications:
+
+  glance:
+    charm: cs:~openstack-charmers-next/glance
+    num_units: 1
+
+  keystone:
+    charm: cs:~openstack-charmers-next/keystone
+    num_units: 1
+
+  mysql:
+    charm: cs:~openstack-charmers-next/percona-cluster
+    num_units: 1
+
+  neutron-api:
+    # Temporary fork until we refactor neutron-arista:
+    charm: cs:~aurelien-lourot/neutron-api-arista-test-fixture
+    num_units: 1
+    options:
+      enable-arista: True
+      enable-ml2-dns: False
+      flat-network-providers: default
+      vlan-ranges: default:1000:3000
+      vni-ranges: 1001:2000
+      neutron-security-groups: True
+      overlay-network-type: vxlan
+      use-internal-endpoints: True
+      enable-l3ha: True
+      dhcp-agents-per-network: 2
+      enable-ml2-port-security: True
+      default-tenant-network-type: vlan
+      l2-population: False
+      ipv4-ptr-zone-prefix-size:
+      reverse-dns-lookup: False
+      dns-domain:
+      manage-neutron-plugin-legacy-mode: True
+
+  neutron-openvswitch:
+    charm: cs:~openstack-charmers-next/neutron-openvswitch
+    num_units: 0
+    options:
+      prevent-arp-spoofing: True
+      firewall-driver: openvswitch
+      enable-local-dhcp-and-metadata: True
+      bridge-mappings: default:br-data
+      data-port: ens7
+
+  nova-cloud-controller:
+    charm: cs:~openstack-charmers-next/nova-cloud-controller
+    num_units: 1
+    options:
+      network-manager: Neutron
+
+  neutron-arista:
+    charm: ../../../neutron-arista
+    num_units: 0
+    options:
+      # Fake IP, will be set to arista-virt-test-fixture's IP address on test
+      # setup:
+      eapi-host: 192.0.2.42
```

Comment:
I like how you have used a IPv4 address from one of the networks reserved for documentation purposes here.